TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java

```diff
@@ -0,0 +1,37 @@
+package tests.data_profiles;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.DashboardPage;
+import page_objects.DataProfilesPage;
+import page_objects.LoginPage;
+import tests.BaseTest;
+import utils.common.Constants;
+import utils.helper.Logger;
+
+public class PositiveDPTests extends BaseTest {
+    LoginPage loginPage = new LoginPage();
+    DataProfilesPage dataProfilesPage = new DataProfilesPage();
+    DashboardPage dashboardPage = new DashboardPage();
+
+    @Test(description = "Verify that all Pre-set Data Profiles are populated correctly")
+    public void tc066_PreSetDataProfilesArePopulatedCorrectly() {
+        Logger.step("Login with a valid user");
+        loginPage.login("administrator", "");
+
+        Logger.step("Go to Data Profiles page.");
+        dashboardPage.clickDataProfilesLink();
+
+        for (int i = 0; i < Constants.PRE_SET_DATA_PROFILES.length; i++) {
+            Logger.step("Verify that Edit and Delete are not displayed at " + Constants.PRE_SET_DATA_PROFILES[i]);
+            Assert.assertNotEquals(dataProfilesPage.getTextOfAction(Constants.PRE_SET_DATA_PROFILES[i]), "Edit", "Edit is available");
+            Assert.assertNotEquals(dataProfilesPage.getTextOfAction(Constants.PRE_SET_DATA_PROFILES[i]), "Delete", "Delete is available");
+
+            Logger.step(String.format("Verify that %s is not clickable.", Constants.PRE_SET_DATA_PROFILES[i]));
+            dataProfilesPage.clickOnPreSetDataProfile(Constants.PRE_SET_DATA_PROFILES[i]);
+
+            Logger.step(String.format("Verify that in front of %s does not have a checkbox.", Constants.PRE_SET_DATA_PROFILES[i]));
```

Comment:
log verify.

Reply:
Sorry, I don't get your idea.

Reply:
It's a verify point so you must use Logger.verify not Logger.step

Reply:
you should use ` Logger.verify` for verify step

Reply:
Oh no. I dont see it. So sorry